utility_scripts/compare_models.py

```diff
@@ -97,16 +99,14 @@ def _create_argument_parser():
                  'must be supplied in either TSV or gzipped TSV format!'
         )
 
-        required.add_argument(
+        optional.add_argument(
             '-l2',
```

Comment:
Consider renaming the short argument to something else. Currently it's not [GNU compliant](https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html) (same for `-s1` & `-s2`).